Lib/annotationlib.py

```diff
@@ -560,6 +565,22 @@ def unary_op(self):
     del _make_unary_op
 
 
+def _template_to_ast(template):
+    values = []
+    for part in template:
+        match part:
+            case str():
+                values.append(ast.Constant(value=part))
+            case _:  # Interpolation, but we don't want to import the string module
+                interp = ast.Interpolation(
+                    value=ast.parse(part.expression),
+                    conversion=ord(part.conversion) if part.conversion is not None else -1,
+                    format_spec=ast.Constant(value=part.format_spec) if part.format_spec != "" else None,
+                )
+                values.append(interp)
```

Comment:
line length / PEP 8
```suggestion
                conversion = part.conversion
                spec = part.format_spec
                value = ast.parse(part.expression)
                conv = ord(conversion) if conversion is not None else -1
                format_spec = ast.Constant(value=spec) if spec != "" else None
                values.append(ast.Interpolation(value=value, conversion=conv,
                                                format_spec=format_spec))
```

Reply:
I think the existing code is clearer.

Reply:
Happy for you to go with something else, but I think for the stdlib we should keep to PEP 8 guidance. Longest line here is over 100 characters.

Reply:
I pushed a different change

Reply:
Thanks! I think line 574 / 578 are still over, the former I think can just move the comment to the line above.

Reply:
I wish we either added an autoformatter or stopped worrying about line lengths. But I split the long lines.